src/style.css

```diff
@@ -1 +1,32 @@
-/* add styles here */
+* {
+  margin: 0;
+}
```

Comment:
Don't use the `*` selector as it impacts performance. It's better to set styles only for elements that require them. If you need to reset margins and paddings, consider using a more specific selector or a CSS reset file.